hw/ip/otp_ctrl/data/otp_ctrl_testplan.hjson

```diff
@@ -0,0 +1,33 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: "otp_ctrl"
+  // TODO: remove the common testplans if not applicable
+  import_testplans: ["hw/dv/tools/testplans/csr_testplan.hjson",
+                     "hw/dv/tools/testplans/mem_testplan.hjson",
+                     "hw/dv/tools/testplans/intr_test_testplan.hjson",
+                     "hw/dv/tools/testplans/tl_device_access_types_testplan.hjson"]
+  entries: [
+    {
+      name: sanity
+      desc: '''
+            Basic sanity test acessing a major datapath within the otp_ctrl.
```

Comment:
nit: sanity -> smoke (there are some more instances of this)

Reply:
Thanks Michael, I believe we have not finalized it within the DV team. Maybe we can leave it as now and once we finalize it, I will make a PR to update all IPs.

Reply:
OK sounds good!